pkg/orchestration/wiring/postgres/plugin.go

```diff
@@ -158,7 +163,7 @@ func (p *WiringPlugin) WireUp(resource *orch_v1.StateResource, context *wiringpl
 
 // instanceParameters constructs ServiceInstance parameters.
 // sharedDbDep may be nil
-func instanceParameters(resource *orch_v1.StateResource, context *wiringplugin.WiringContext, sharedDbDep *wiringplugin.WiredDependency) ([]byte, bool /* externalErr */, bool /* retriable */, error) {
+func (p *WiringPlugin) instanceParameters(resource *orch_v1.StateResource, context *wiringplugin.WiringContext, sharedDbDep *wiringplugin.WiredDependency) ([]byte, bool /* externalErr */, bool /* retriable */, error) {
```

Comment:
@nilebox I opted for a receiver param over an additional function param to avoid clashing with your plans in this place. I would prefer a function param if it doesn't get in the way.

Reply:
that's fine